.github/workflows/pull_request_integration_tests_vm_5.15_x86_64.yml

```diff
@@ -0,0 +1,14 @@
+name: Pull request integration tests on VM 5.15 x86_64
```

Comment:
As discussed offline, this test is failing due to unrelated bugs. Should we fix the bugs before merging this, or should we disable this kernel version for now, and re-enable later after the bug is fixed?

Reply:
I think we should fix before we merge this test.